debian/control

```diff
@@ -34,7 +34,7 @@ Build-Depends:
    python3-lxml,
    python3-defusedxml,
    python3-xmlschema (>= 1.10.0),
-   python3-elementpath (>= 2.5.0)
+   python3-elementpath (>= 2.5.0),
```

Comment:
@junjiemao1 @Nanlin the element path version should be = 2.5.0 (and not greater) right?